### ISSUE TITLE
Client sometimes disconnects from host with the Pause in Multiplayer.

### ISSUE THREAD

**Issue by brucethemoose** [jorgamun/PauseInMultiplayer#2]
This error shows up in the server when it happens: `[22:19:52 ERROR game] Farmhand Spencer does not have Pause in Multiplayer mod.`

And I sometimes get this as well: 
```
[22:42:42 ERROR Pause in Multiplayer] This mod failed in the GameLoop.UpdateTicking event. Technical details: 
NullReferenceException: Object reference not set to an instance of an object.
   at PauseInMultiplayer.PauseInMultiplayer.GameLoop_UpdateTicking(Object sender, UpdateTickingEventArgs e) in C:\Users\William\source\repos\PauseInMultiplayer\PauseInMultiplayer\PauseInMultiplayer.cs:line 541
   at StardewModdingAPI.Framework.Events.ManagedEvent`1.Raise(TEventArgs args, Func`2 match) in SMAPI\Framework\Events\ManagedEvent.cs:line 126
```

Full host log: https://smapi.io/log/f41921693b9b4a9bacfc6fae204ffb8d

Don't have a disconnected client's log yet, will post it when I do.

**Comment by jorgamun** [comment]
That's a tricky one. As soon as you get the SMAPI log for Spencer, post it here, as that would help a lot in tracking down what's causing this. Thanks Bruce!

**Comment by brucethemoose** [comment]
Will do, I just wish I knew how to repro it. It happened maybe three times before, but it (fortunately(?)) hasn't happened since then in a very long play session.

**Comment by brucethemoose** [comment]
Hey, we took a break for a bit and updated SMAPI and some core mods, but we still get some rare Pause in Multiplayer errors, mostly during events! 

Here's an issue we ran while the host was in the final community center cutscene, and the client was in a ridgeside village cutscene. The client desynced when their cutscene was over, and had to rejoin.

Server log: https://smapi.io/log/b32e85f5e84d49c89c822466101e68b9

Client log: https://smapi.io/log/0274e1915e92457ab93cb3fdc4568a0c
